src/System.Management.Automation/resources/Modules.resx

```diff
@@ -606,4 +606,7 @@
   <data name="PrerequisiteForDesktopEditionOnly" xml:space="preserve">
     <value>This prerequisite is valid for the PowerShell Desktop edition only.</value>
   </data>
+  <data name="PSEditionNotSupported" xml:space="preserve">
+    <value>Module '{0}' does not support current PowerShell edition '{1}'. Its supported editions are '{2}'. Use 'Import-Module -SkipCompatibilityCheck' to ignore the compatibility of this module.</value>
```

Comment:
Should we use imperative mood in ` Use 'Import-Module -SkipCompatibilityCheck' `? 
Also `{0}` can be long path.